src/components/InteriorLeftNav/InteriorLeftNav.js

```diff
@@ -13,22 +13,30 @@ export default class InteriorLeftNav extends Component {
     className: PropTypes.string,
     activeHref: PropTypes.string,
     onToggle: PropTypes.func,
+    initialOpenState: PropTypes.bool,
   };
 
   static defaultProps = {
     onToggle: () => {},
+    initialOpenState: true,
   };
 
   state = {
     activeHref:
       this.props.activeHref || (window.location && window.location.pathname),
-    open: true,
+    open:
+      this.props.initialOpenState !== undefined
+        ? this.props.initialOpenState
+        : true,
   };
 
   componentWillReceiveProps = nextProps => {
     if (nextProps.activeHref) {
       this.setState({ activeHref: nextProps.activeHref });
     }
+    if (nextProps.initialOpenState !== undefined) {
```

Comment:
The condition should be `this.props.open !== nextProps.open`, meaning, there is the change in the corresponding prop.